example_app_generator/generate_app.rb

```diff
@@ -20,6 +20,7 @@
   gsub_file 'Gemfile', /^.*\bgem 'rails.*$/, ''
   gsub_file "Gemfile", /.*web-console.*/, ''
   gsub_file "Gemfile", /.*debugger.*/, ''
+  gsub_file "Gemfile", /.*puma.*/, ""
```

Comment:
this deletes the generated puma requirement from the default rails generator, replacing it with ours.